components/ui/Tag.jsx

```diff
@@ -0,0 +1,9 @@
+const Tag = ({ content }) => {
+  return (
+    <p className="font-neuf text-lg text-primaryGray-400  -tracking-wide  bg-primaryGray-200  rounded-base  px-2.5  py-sm w-fit">
```

Comment:
Remove empty spaces on this line

```suggestion
    <p className="font-neuf text-lg text-primaryGray-400 -tracking-wide bg-primaryGray-200 rounded-base px-2.5 py-sm w-fit">
```